src/main/java/frc/robot/subsystems/vision/LimelightLogInputs.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems.vision;
+
+import org.littletonrobotics.junction.AutoLog;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+//@AutoLog
+//public class LimelightLogInputs {
+//    public double targetDistance = 0;
+//    public double yaw = 0;
+//    public boolean hasTargets = false;
+//    public float tagId = 0;
+//}
+
+public class LimelightLogInputs implements LoggableInputs {
+    public double targetDistance = 0;
+    public double yaw = 0;
+    public boolean hasTargets = false;
+    public float tagId = 0;
+
+    public LimelightLogInputs() {
+    }
```

Comment:
The constructor is unnecessary, Java will automatically generate empty public constructor for you